env/build_hera_intel.env

```diff
@@ -2,42 +2,46 @@
 
 module purge
 
+module load cmake/3.16.1
+module load hpss
 module use /contrib/sutils/modulefiles
 module load sutils
-module load cmake/3.16.1
 
-module use /scratch2/NCEPDEV/nwprod/hpc-stack/test/modulefiles/stack
-
-module load hpc/1.0.0-beta1
+### hpc-stack ###
+module use /scratch2/NCEPDEV/nwprod/hpc-stack/libs/hpc-stack/modulefiles/stack
+module load hpc/1.1.0
 module load hpc-intel/18.0.5.274
 module load hpc-impi/2018.0.4
-module load jasper/2.0.22
-module load zlib/1.2.11
-module load png/1.6.35
-module load hdf5/1.10.6
-module load netcdf/4.7.4
-module load pio/2.5.1
-module load esmf/8_1_0_beta_snapshot_27
+
 module load bacio/2.4.1
 module load crtm/2.3.0
+module load gfsio/1.4.1
 module load g2/3.4.1
-module load g2tmpl/1.9.1
+module load g2tmpl/1.10.0
 module load ip/3.3.3
+module load jasper/2.0.22
 module load nemsio/2.5.2
+module load png/1.6.35
+module load sfcio/1.4.1
+module load sigio/2.3.2
 module load sp/2.3.3
-module load w3emc/2.7.3
+module load wgrib2/2.0.8
 module load w3nco/2.4.1
-module load upp/10.0.0
+module load zlib/1.2.11
 
-module load gfsio/1.4.1
-module load sfcio/1.4.1
+### Additional modules ###
+module load esmf/8_1_0_beta_snapshot_27
```

Comment:
@mkavulich, the latest ufs_utils still uses esmf/8_1_0_beta_snapshot_27 (https://github.com/NOAA-EMC/UFS_UTILS/blob/develop/modulefiles/build.hera.intel).

Reply:
@mkavulich, does the dycore use bs47, but UFS_UTILS still uses 27?  Have these two repos used different versions of ESMF in the past?  How is this handled?  I'm wondering if the problems with build that we're observing currently are related to the ESMF version.

Reply:
That all makes sense... I was asking because of the update to hpc/1.1.0, I thought that change was made in the weather model at the same time esmf was updated but I was mistaken. Thank you for addressing my other concern as well. I am okay with this PR being merged, though I do plan on making further updates to the modules soon; I will be sure to check that UFS_UTILS works with the newer updates.